lotus-soup/go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/multiformats/go-multiaddr-net v0.2.0
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
 	github.com/stretchr/objx v0.2.0 // indirect
-	github.com/testground/sdk-go v0.2.6-0.20201013134225-044f6f8ec366
```

Comment:
Bumping up the `sdk-go` since @hacdias fixed the interfaces on the sidecar and testground side.